sympy/physics/control/lti.py

```diff
@@ -1422,3 +1435,510 @@ def _eval_rewrite_as_TransferFunction(self, num, den, **kwargs):
+    >>> pprint(tfm_7, use_unicode=False)
+    [ 1      1   ]
+    [----    -   ]
+    [   2    s   ]
+    [s*s         ]
+    [            ]
+    [ 1    1    1]
+    [ --   -- + -]
+    [  2    2   s]
+    [ s    s     ]
+    >>> tfm_7.doit()
+    TransferFunctionMatrix(((TransferFunction(1, s**3, s), TransferFunction(1, s, s)), (TransferFunction(1, s**2, s), TransferFunction(s**2 + s, s**3, s))))
+    >>> pprint(_, use_unicode=False)
+    [1     1   ]
+    [--    -   ]
+    [ 3    s   ]
+    [s         ]
+    [          ]
+    [     2    ]
+    [1   s  + s]
+    [--  ------]
+    [ 2     3  ]
+    [s     s   ]
+
+    See Also
+    ========
+
+    TransferFunction, Series, Parallel, Feedback
+
+    """
+    def __new__(cls, arg):
+
+        expr_mat_arg = []
+        try:
+            var = arg[0][0].var
+        except TypeError:
+            raise ValueError("`arg` param in TransferFunctionMatrix should "
+            "strictly be a nested list containing TransferFunction objects.")
+        for row_index, row in enumerate(arg):
+            temp = []
+            for col_index, element in enumerate(row):
+                if not isinstance(element, (TransferFunction, Series, Parallel)):
+                    raise TypeError("Incompatible type found as the element of "
+                    f"TransferFunctionMatrix at index {(row_index, col_index)}.")
+
+                if var != element.var:
+                    raise ValueError("Conflicting value(s) found for `var`. All TransferFunction instances in "
+                            "TransferFunctionMatrix should use the same complex variable in Laplace domain.")
+
+                temp.append(element.to_expr())
+            expr_mat_arg.append(temp)
+
+
+        if isinstance(arg, (list, Tuple)):
+            # Making nested Tuple (sympy.core.containers.Tuple) from nested list or nested python tuple
+            arg = Tuple(*(Tuple(*r, sympify=False) for r in arg), sympify=False)
+
+        obj = super(TransferFunctionMatrix, cls).__new__(cls, arg)
+        obj._expr_mat = ImmutableMatrix(expr_mat_arg)
+        return obj
+
+    @classmethod
+    def from_Matrix(cls, matrix, var):
+        """
+        Creates a new ``TransferFunctionMatrix`` efficiently from a SymPy Matrix of ``Expr`` objects.
+
+        Parameters
+        ==========
+
+        matrix : ``ImmutableMatrix`` having ``Expr``/``Number`` elements.
+        var : Symbol
+            Complex variable of the Laplace transform which will be used by the
+            all the ``TransferFunction`` objects in the ``TransferFunctionMatrix``.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s
+        >>> from sympy.physics.control.lti import TransferFunctionMatrix
+        >>> from sympy.matrices import Matrix
+        >>> from sympy.printing import pprint
```

Comment:
Both `Matrix` and `pprint` can be imported as 
```
from sympy import Matrix, pprint
```
In the docs we should recommend that top-level names be imported in this way because we don't want users to import directly from submodules like `sympy.matrices`.

Reply:
This comment applies to all of the docstrings